src/core/lib/security/credentials/xds/xds_channel_creds.h

```diff
@@ -0,0 +1,97 @@
+//
```

Comment:
These two files can be moved to the parent directory.  There's no need for an "xds" subdirectory here.

Reply:
Moved.

Reply:
Probably I messed up while rebasing and I force-pushed, please see [fc428bd](https://github.com/grpc/grpc/pull/28746/commits/fc428bd6b03219b986520d8f27b1c2ca22c073ad) that has all changes from the last iteration.